independent-publisher-connectors/ChuckNorris.io/apiDefinition.swagger.json

```diff
@@ -0,0 +1,193 @@
+                  "type": "string",
+                  "description": "icon_url",
+                  "title": "Icon URL",
+                  "x-ms-visibility": "internal"
+                },
+                "id": {
+                  "type": "string",
+                  "description": "id",
+                  "title": "ID",
+                  "x-ms-visibility": "internal"
+                },
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "title": "URL",
+                  "x-ms-visibility": "internal"
+                },
+                "value": {
+                  "type": "string",
+                  "description": "value",
+                  "title": "Value"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get random Chuck Norris fact",
+        "description": "This action will give you a random Chuck Norris fact",
+        "operationId": "GetRandomChuckNorrisFact",
+        "parameters": [
+          {
+            "name": "category",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Select a category",
+            "x-ms-dynamic-values": {
+              "operationId": "GetCategories"
+            },
+            "x-ms-summary": "Category"
+          }
+        ]
+      }
+    },
+    "/categories": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "title": "Categories"
+              }
+            }
+          }
+        },
+        "summary": "Get Chuck Norris fact categories",
+        "description": "This action will get all the possible categories of Chuck Norris facts",
+        "operationId": "GetCategories",
+        "parameters": [],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/search": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "total": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "total",
+                  "title": "Total",
+                  "x-ms-visibility": "internal"
+                },
+                "result": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "categories": {
+                        "type": "array",
+                        "items": {},
+                        "description": "categories"
+                      },
+                      "created_at": {
+                        "type": "string",
+                        "description": "created_at",
```

Comment:
@Laskewitz Can you please update it to `Created at`.

Documentation - [Description values of properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the similar description values in the swagger.